### PR TITLE
BugFix: Fix Converting null literal in ServiceCollectionExtensions

### PR DESCRIPTION
**Pull request by SvyatoS1**

### CODE REVIEW

**Review by MrVenik — COMMENTED** [review]
Maybe using null checking and throwing another exception with message that would describe what exactly is going wrong - would be more preferable ?

**Comment by SvyatoS1** [comment]
> Maybe using null checking and throwing another exception with message that would describe what exactly is going wrong - would be more preferable ?

Okay, i'll fix that. I just thinking about in how many cases 'DefaultConnection' will be null, and to simpify use !, but we can also implement null error handling.

**Comment by MrVenik** [comment]
> > Maybe using null checking and throwing another exception with message that would describe what exactly is going wrong - would be more preferable ?
> 
> Okay, i'll fix that. I just thinking about in how many cases 'DefaultConnection' will be null, and to simpify use !, but we can also implement null error handling.

Or just check if connection string is null and throw exception.
Only if user forgot to add connection strings to user secrets file can lead us to this exception, the exception would not be handled at all, because it is critical error.

File: Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -25,6 +25,7 @@
 using Streetcode.BLL.Services.Text;
 using Serilog.Events;
 using FluentValidation;
+using static Microsoft.EntityFrameworkCore.DbLoggerCategory.Database;
```
Comment (MrVenik) [inline Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs:28]: Is this using needed ?

**Review by MrVenik — CHANGES_REQUESTED** [review]

File: Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs (hunk 1 above)
Comment (SvyatoS1) [reply Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs]: Maybe not, it was added automatically, in last commit i remove it.

**Review by MrVenik — APPROVED** [review]

**Review by Skilskyi — APPROVED** [review]

**Pull request merged**